components/speedSections.py

```diff
@@ -0,0 +1,41 @@
+import logging as log
+from utils.configMapper import findConfig
+from utils import yaml
+from robotMap import RobotMap
+import os
+class SpeedSections():
+    configPath = os.path.join(findConfig()[1],"speedSections.yml")

```

Comment:
What happens here if findConfig()[1] fails? Can we at least throw a good error about needing the speedSections.yml file?